Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER Code Climate <hello@codeclimate.com>
 
 RUN adduser -u 9000 -D app
-USER app
```

Comment:
This is in the spec and should be preserved

> A non-root user named app must be created with UID and GID 9000 and declared using the USER directive.

https://github.com/codeclimate/spec/blob/master/SPEC.md#packaging